api/api.go

```diff
@@ -18,10 +18,10 @@ import (
 )
 
 var (
-	createPermission = auth.Permissions{Create: true}
+	// createPermission = auth.Permissions{Create: true}
 	readPermission   = auth.Permissions{Read: true}
 	updatePermission = auth.Permissions{Update: true}
-	deletePermission = auth.Permissions{Delete: true}
+	// deletePermission = auth.Permissions{Delete: true}
 )
 
```

Comment:
Why are these lined commented out?

Reply:
I introduced a nolint comment for this to pass the make lint command. Is this ok ?